README.md

```diff
@@ -75,6 +75,21 @@ During development, you can use the following command to clean the database and
 sudo -u postgres psql -d geoengine -c "drop schema public cascade; create schema public authorization geoengine; create extension postgis;" && cargo run --features pro
 ```
 
+##### NFDI / GFBio / GBIF
+
+For running the NFDI / GFBio / GBIF data providers during development, you need to integrate the test data like this:
+
+```bash
+sudo -u postgres psql --dbname=geoengine -c \
```

Comment:
maybe make subsections for each provider? Or at least say which line is relevant for which provider